src/App.css

```diff
@@ -0,0 +1,66 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 40px;
+}
+
+.header_logo {
+  width: 180px;
+  height: 78px;
+}
+.header_user {
+  width: 90px;
+  height: 90px;
+}
+.wrapper_search {
+  float: right;
+  margin-bottom: 40px;
+  margin-top: 90x;
+  margin-top: -6%;
+  margin-right: 80px;
+}
+.wrapper_logo {
+  padding: 20px;
+}
+
+form > input {
+  border: none;
+  background: transparent;
+  margin: 0;
+  padding: 10px;
+  font-size: 14px;
+  color: inherit;
+  border: 1px solid grey;
+  border-radius: 5px;
+}
+
+input[type="search"]::placeholder {
```

Comment:
Use html tag- css selectors on the top of your code, it will prevent CSS specificity collisions